documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,369 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, known issues, and resolved issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
```

Comment:
@HagayVider1 @anarnold97 we also need to document that must-gather logs cannot be downloaded from the UI anymore (so it needs to be done through the CLI), see [internal conversion about that with @yaacov ](https://redhat-internal.slack.com/archives/C049WBVTRD2/p1712561122500229?thread_ts=1712507636.418859&cid=C049WBVTRD2)

Reply:
Ref: https://issues.redhat.com/browse/MTV-946?focusedId=24491851&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24491851

Reply:
See lines 45-4t at end of Technical changes